strongbox-storage/strongbox-storage-core/src/main/java/org/carlspring/strongbox/services/ConfigurationManagementService.java

```diff
@@ -96,4 +97,10 @@ boolean removeRepositoryArtifactCoordinateValidator(String storageId,
 
     void setSmtpSettings(MutableSmtpConfiguration smtpConfiguration) throws IOException;
 
+    /**
+     * Attempts to set max file upload size globally
+     * if global MultipartConfigElement is set to instance of UpdatableMultipartConfigElement
+     * the change is set successfully and method returns true, returns false otherwise
+     * */
```

Comment:
```suggestion
    /**
     * Sets the max file upload size globally,
     * if global MultipartConfigElement is set to instance of UpdatableMultipartConfigElement,
     * the change is set successfully and method returns true, returns false otherwise.
     * */
```